app/services/api/v1x1/catalog.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1x1
+    module Catalog
+      class AddToOrder     < Api::V1x0::Catalog::AddToOrder; end
+      class CancelOrder    < Api::V1x0::Catalog::CancelOrder; end
+      class CreateIcon     < Api::V1x0::Catalog::CreateIcon; end
+      class DuplicateImage < Api::V1x0::Catalog::DuplicateImage; end
+      class SoftDelete     < Api::V1x0::Catalog::SoftDelete; end
```

Comment:
This list seems small, are these just the ones we currently use in v1x1 controllers?